tests/main/dirs-not-shared-with-host/task.yaml

```diff
@@ -26,15 +27,16 @@ environment:
     DIRECTORY/alternatives: /etc/alternatives
     DIRECTORY/ssl: /etc/ssl
 execute: |
+    #shellcheck source=tests/lib/dirs.sh
     . "$TESTSLIB/dirs.sh"
     echo "We can check the inode number of $DIRECTORY"
-    host_inode="$(stat -c '%i' $DIRECTORY)"
```

Comment:
Why is shellcheck complaining here? The `var=$(...)` assignment does not need quoting but does it hurt to add it?

Reply:
It complained about `$DIRECTORY` not being quoted.